test/e2e_bind_test.go

```diff
@@ -1,13 +1,34 @@
 package test
 
 import (
+	"fmt"
 	"net/http"
+	"os"
+	"os/exec"
 	"testing"
+	"time"
 )
 
-func TestE2EBind(t *testing.T) {
-	req, err := http.NewRequest(http.MethodGet, "http://localhost:10000/", nil)
+func startEnvoy(configPath string) {
+	cmd := exec.Command("envoy", "-c", configPath)
```

Comment:
we could just accept peices of the http filter configurations to generate a whole config file, instead of passing a full config file name.
that means we don't need a envoy config file for each test case.

Reply:
OK